README.md

```diff
@@ -1,2 +1,22 @@
-# visualize-algorithms
-:fire: Visualize Algorithms
+<h1 align="center">Visualize Algorithms</h1>
+ <br>
+ <img src="https://github.com/SarahLizDettloff/visualize-algorithms/blob/master/screenshot.png?raw=true" alt="Visualize Algorithms"> </br>
+
+---
+## About
+A tool which provides data visualization to algorithms. Current features:
+* Selection Sort
+* Bubble Sort
+* Optimized Bubble Sort
+* Merge Sort
+
+## Clone Directions
+``` git clone https://github.com/tanwanimohit/visualize-algorithms.git ```
+
+
+
+## Contrbutors
```

Comment:
Remove Contributors info, as anyone can see that in insights.

Reply:
Contributors section has been removed. Please see newest commit.